atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -10,9 +10,17 @@
 try:
     # py2
     from ConfigParser import ConfigParser
+    # Although io.StringIO is available in python2, it
+    # always expects unicode during write. Since
+    # ConfigParser.ConfigParser does not support
+    # writing unicode, io cannot be used.
+    from StringIO import StringIO
```

Comment:
I think the trick was to read via ```BytesIO``` and write using ```StringIO``` due to a bug in py3, see ```pre_add_yum_repo_by_url.py``` code.

OTOH this is counter-intuitive, so probably we should keep StringIO